test/ably/resttoken_test.py

```diff
@@ -179,6 +180,12 @@ def test_toke_details_from_json(self):
             TokenDetails.from_json(token_details_str),
         )
 
+    # Issue #71
+    @dont_vary_protocol
+    def test_request_token_float(self):
+        lifetime = datetime.timedelta(hours=4)
+        self.ably.auth.request_token({'ttl': lifetime.total_seconds() * 1000})
```

Comment:
It's ms. That's what the spec asks for, no?

    (RSA5) TTL for new tokens is specified in milliseconds.
    (RSA9e) TTL is optional and specified in milliseconds.
    (TK2a) ttl long – Requested time to live for the token in milliseconds.